bitnami/metallb/Chart.yaml

```diff
@@ -28,4 +28,9 @@ sources:
   - https://github.com/metallb/metallb
   - https://github.com/bitnami/containers/tree/main/bitnami/metallb
   - https://metallb.universe.tf
-version: 4.1.10
+version: 4.1.11
+
+# The optional kubeVersion field can define semver constraints on supported Kubernetes versions.
+# Helm will validate the version constraints when installing the chart and fail if the cluster
+# runs an unsupported Kubernetes version.
+kubeVersion: ">= 1.19.0-0"
```

Comment:
@javsalgar Should we consider it for all charts?

Reply:
We are already adding it to those Helm charts where there is a specific k8s resource that was deprecated in a specific Kubernetes version:
```console
$ ag 'kubeVersion' */Chart.yaml
contour-operator/Chart.yaml
20:kubeVersion: '>= 1.21.0-0'

kubeapps/Chart.yaml
27:kubeVersion: ">=1.21.0-0"

kube-prometheus/Chart.yaml
28:kubeVersion: '>= 1.16.0-0'

nginx-ingress-controller/Chart.yaml
22:kubeVersion: '>= 1.20.0-0'

rabbitmq-cluster-operator/Chart.yaml
21:kubeVersion: '>= 1.19.0-0'

sealed-secrets/Chart.yaml
18:kubeVersion: '>=1.16.0-0'
```